packages/components/src/tabs/styles.ts

```diff
@@ -251,11 +251,7 @@ export const TabChevron = styled( Icon )`
 	// indicator that's being animated at the same time.
 	@media not ( prefers-reduced-motion ) {
 		[data-select-on-move='true']
-			[role='tab']:is(
-				[aria-selected='true'],
-				[data-focus-visible],
-				:hover
-			)
+			[role='tab']:is( [aria-selected='true'],  )
 			& {
 			transition: opacity 0.3s ease-in;
```

Comment:
This works well enough so feel free to disregard this comment. It’s super subtle but I think a small delay is better. Mechanically the delay should match the transition duration of the indicator (`0.2s` I think). However, to make sure it doesn’t feel like the icon appearance is late the delay can be less than that. Here’s my suggestion:
```css
transition: opacity 0.15s 0.15s linear;
```
I also recommend `linear` easing because on a quick `opacity` transition `ease-in` is imperceptible (though without the delay it might helpful in this case).

Reply:
We actually tried a few things including delay and different easings, and the current one felt decent. However if you've tested this value live and feel like it's an improvement, I'm happy to change it, since we don't feel too strongly about it anyway :)

Let me know! Really appreciate your input 👍

Reply:
Thanks! I did test but honestly won’t argue it’s a significant improvement. It works as is. Non-linear easing on opacity or color transitions always invites my skepticism but in this case `ease-in` is a pseudo delay.

Reply:
Forgot to say, but I implemented your transition parameters. Thank you again!